Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Development Dockerfile for bioformats2raw
+# -----------------------------------------
+
+# To install the built distribution into other runtimes
+# pass a build argument, e.g.:
+#
+#   docker build --build-arg IMAGE=openjdk:9 ...
+#
+
+# Similarly, the BUILD_IMAGE argument can be overwritten
+# but this is generally not needed.
+ARG BUILD_IMAGE=gradle:5.2.1-jdk8
+
+#
+# Build phase: Use the gradle image for building.
+#
+FROM ${BUILD_IMAGE} as build
+USER root
+RUN apt-get update -qq && apt-get install -y -qq zeroc-ice-all-dev libblosc-dev
+RUN echo Temporary && apt-get install -y -qq git maven
+RUN mkdir /bioformats2raw && chown 1000:1000 /bioformats2raw
+
+# Build all
+USER 1000
+
+COPY --chown=1000:1000 . /bioformats2raw
+WORKDIR /bioformats2raw
+RUN echo Temporary \
```

Comment:
I assume there's a good reason for doing this rather than just depending on the n5-zarr 0.0.2-beta artifact?

Reply:
Yeah: GitHub was failing on that day. :smile: I'll revert.